app/models/task.rb

```diff
@@ -1,5 +1,9 @@
 require 'sqlite3'
 
+
+
+
+
```

Comment:
Was hoping adding these spaces would allow me to comment on what's below... no such luck.

We talked about this today, but I figured I'd make a comment. Lines 23/24 and 32/33 could be removed and we can just use `@database` or even `database`.

I really like that you have an example code base for them to reference 🎉